Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN cargo build --locked --release --package sui
 FROM debian:bullseye-slim AS base
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends curl ca-certificates procps vim sudo awscli && \
+    apt-get install -y --no-install-recommends curl ca-certificates procps vim sudo awscli jq && \
```

Comment:
I missed when all these got bundled into the base image ... We should try to keep these images minimal and cloud neutral. For one-off debugging use cases, we can always build custom images off of this one. I suggest taking out `vim`, `sudo`, `awscli`, `jq`.

Reply:
`vim` and `sudo` aren't needed anymore, I can take those out. `awscli` and `jq` will be needed for snapshot restoration (downloading snapshots, processing aws api responses`

Reply:
> `awscli` and `jq` will be needed for snapshot restoration (downloading snapshots, processing aws api responses, since right now snapshots only work with aws s3)

- Do they have to be baked into the sui node image? Can we use a separate init image to do the restoration?
- If they must go into the sui node image, I still suggest building another custom private image off of this image.